more_autograding_examples/cpp_updated_random_input_output/config/config.json

```diff
@@ -0,0 +1,15 @@
+{
+	"required_capabilities": "cpp",
+    "instructor_solution_compile_command": "g++ -std=c++11 -Wall -Wextra",
+    "testcases" : [
```

Comment:
Let's add in a field to the config file to specify the name of the instructor_executable_name (similar to the compilation test executable_name field).  For full flexibility, both the compile command and the executable names should be strings or arrays of strings (again this is already allowed for commands and executable names in compilation tests) -- to allow the instructor to build multiple solution executables.

This will make some of code in the build_homework_function.sh file less hardcoded/fragile.

Reply:
Exactly, this is what I planned to discuss in the meeting. Will implement it 
and I suppose, In the case of array of strings : Index of the compile command corresponds to the index of the name of instructor_executable during the implementation.